src/pac.c

```diff
@@ -435,6 +435,43 @@ pac_stash_decayed(tsdn_t *tsdn, pac_t *pac, ecache_t *ecache,
 	return nstashed;
 }
 
+static bool
+decay_with_process_madvise(edata_list_inactive_t *decay_extents) {
+	cassert(have_process_madvise);
+	assert(opt_process_madvise_max_batch > 0);
+#ifndef JEMALLOC_HAVE_PROCESS_MADVISE
+	return true;
+#else
+	size_t max_stack = VARIABLE_ARRAY_SIZE_MAX / sizeof(struct iovec);
+	size_t len = max_stack < opt_process_madvise_max_batch ? max_stack :
+	    opt_process_madvise_max_batch;
+	struct iovec vec[len];
```

Comment:
This is VLA? Will it affect the compatibility for some C++ compilers?

Reply:
Good catch. Replaced with `VARIABLE_ARRAY`